examples/games/padracing/lib/padracing_game.dart

```diff
@@ -58,13 +57,13 @@ class PadRacingGame extends Forge2DGame with KeyboardEvents {
 
   @override
   Future<void> onLoad() async {
+    super.onLoad();
+    cameraComponent.removeFromParent();
```

Comment:
I imagine this is because it gets re-added later and you want to call onLoad to reset the game? just curious what breaks if you just leave the camera there as is?

Reply:
This camera isn't used, it uses several other cameras later instead.